index.js

```diff
@@ -32,8 +32,8 @@ app.get('/', (req, res)=>{
 app.get('/register', (req, res)=>{
     const con = mysql.createConnection({
         host: "localhost",
-		user: "Daniel",
-		password: "pass",
```

Comment:
Leave this as "Daniel" and "pass" ill likely change it to a more universal name.